frontend/src/profiles/uvp/reports/activity-report/activity-report.component.html

```diff
@@ -112,6 +112,7 @@
                             <button
                                 mat-menu-item
                                 (click)="openDataset(element)"
+                                [disabled]="element.state == 'delete'"
```

Comment:
`[disabled]="element.deleted`

Refactor this so the logic is kept out of the template.